### PR TITLE
CB-688. README additions for Linux development.

### PR DESCRIPTION
**Pull request by meowfaceman**
Additions have been made to the README.md to clarify some things
necessary specific to Linux development.

Note: This is pretty minor -- a lot of the changes made recently have made it nicer to work on Linux, since the Mac experience and the Linux experience are closer together.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,6 +88,17 @@ If everything went well then Cloudbreak will be available on http://localhost. F
 
 The deployer has generated a `certs` directory under `cbd-local` directory which will be needed later on to set up IDEA properly.
 
+### Linux differences and obtaining the docker bridge address
+
+The profile section needs two extra configuration options:
+
+```
+export PUBLIC_IP=127.0.0.1
+export CB_LOCAL_DEV_BIND_ADDR=<DOCKER_BRIDGE_ADDRESS>
```
Comment (mhmxs) [inline README.md]: I haven't find `CB_LOCAL_DEV_BIND_ADDR` in the latest deployer.

File: README.md [hunk 2]
```diff
@@ -117,12 +135,11 @@ IntelliJ IDEA -> Preferences -> Editor -> Inspections -> Settings icon -> Import
...
 
-
 To launch the Cloudbreak application execute the `com.sequenceiq.cloudbreak.CloudbreakApplication` class  (Set 'Use classpath of module' to `core_main`) with the following VM options:
 ```
 -Dcb.client.id=cloudbreak
 -Dcb.client.secret=CB_SECRET_GENERATED_BY_CBD
--Dcb.db.port.5432.tcp.addr=localhost
+-Dcb.db.port.5432.tcp.addr=<DB_ADDRESS>
 -Dcb.db.port.5432.tcp.port=5432
 -Dcb.identity.server.url=http://localhost:8089
```
Comment (mhmxs) [inline README.md]: This should be also the `localhost` os `bridge ip`

Comment (meowfaceman) [reply README.md]: Forgive me, which line? Line 142 is DB_ADDRESS and is explained by a blurb below it to be localhost or bridge ip.

Comment (mhmxs) [reply README.md]: `-Dcb.identity.server.url=http://localhost:8089`

Comment (meowfaceman) [reply README.md]: This actually is not necessary -- only the database needs to use the docker bind address, nothing else.

**Pull request merged**